src/app/clusters/messages-server/messages-server.h

```diff
@@ -0,0 +1,29 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/util/basic-types.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace Messaging {} // namespace Messaging
```

Comment:
is there any use having these empty namespaces here?

Reply:
it's a copy pasta from https://github.com/project-chip/connectedhomeip/pull/30254/. However as I plan to enable this cluster similarly to the previous clusters, this name space will be extended with some methods.